docs/GCP-development.md

```diff
@@ -341,10 +362,10 @@ gcr.io/chainguard-dev/pythontest@sha256:c089acd03a21830c329d70f61cefa2a29c43e59e
 
 Get the SBOM:
 
-Pull down the Root CA from fulcio , Make sure you have set up [Network Access](#network)
+Pull down the Root CA from fulcio, Make sure you have set up Network Access
```

Comment:
nit: fulcio => Fulcio
We can go through these later with tooling, but if you make other changes while here :)